djstripe/utils.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+"""Utility functions related to the djstripe app."""
```

Comment:
full module docstring please

authors are me, pydanny, and wahuneke 
https://github.com/kavdev/dj-stripe/blame/%23162-api-updates-through_2015-07-28/djstripe/utils.py